Monolog/Handler/SlackHandler.php

```diff
@@ -11,16 +11,18 @@
 
 /**
  * @internal
+ *
+ * @phpstan-type Level Logger::DEBUG|Logger::INFO|Logger::NOTICE|Logger::WARNING|Logger::ERROR|Logger::CRITICAL|Logger::ALERT|Logger::EMERGENCY
```

Comment:
import from https://github.com/Seldaek/monolog/blob/main/src/Monolog/Logger.php#L30
```suggestion
 * @phpstan-import-type Level from \Monolog\Logger
```